member_database/events/models.py

```diff
@@ -12,6 +13,16 @@ class Event(db.Model):
     notify_email = db.Column(db.Text)
     force_tu_mail = db.Column(db.Boolean, default=False)
 
+    
+    def default_shortlink(context):
+        """Generates a default shortlink out of the name of the event, by removing spaces and making it URL-safe."""
+        if context is not None:
+            return quote(context.get_current_parameters()["name"].replace(" ", ""))
+        else:
+            return ""
+
+    shortlink = db.Column(db.Text, default=default_shortlink)
```

Comment:
I'd remove the default and add a unique constraint

Reply:
I'll add the unique constraint.

Reply:
My suggestion: Don't create the default names in migration, since this is not necessary compatible to the unique constraint. 
But let the default shortlink still be created. This only adds features, not remove any. If you want a non-default shortlink, just type it in the admin view!

Reply:
@maxnoe what do you think?